app/src/main/res/layout/item_solana_deposit.xml

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
are you sure it's `item_solana_deposit`? :) 
`item_solend_deposit`?

Reply:
heh =) sure